tests/wicked/advanced/sut/bridge_interface_legacy.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Advanced test cases for wicked
+# Test 11: Create Bridge interface from legacy ifcfg files
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base 'wickedbase';
+use strict;
+use testapi;
+use utils 'systemctl';
+use lockapi;
+use mmapi;
+
+sub run {
+    my ($self) = @_;
+    my $config = '/etc/sysconfig/network/ifcfg-br0';
+    my $dummy  = '/etc/sysconfig/network/ifcfg-dummy0';
+    record_info('Info', 'Create Bridge interface from legacy ifcfg files');
+    $self->get_from_data('wicked/ifcfg/br0',    $config);
+    $self->get_from_data('wicked/ifcfg/dummy0', $dummy);
+    $self->setup_bridge($config, $dummy, 'ifup');
+    my $res = $self->get_test_result("br0", "");
```

Comment:
Second parameter of get_test_result() isn't necessary. If you going to change it, the other files have it too.
Maybe change to  single quote.